configs/regenerate.sh

```diff
@@ -1,13 +1,24 @@
 #!/usr/bin/env bash
-
-binaries=(server connector translator addzone webconfig rollback keymaker poisonrecordmaker authmanager rotate)
+#
+# Regenerate example configuration files.
+#
+# Run as
+#
+#     ./configs/regenerate.sh [dst-dir]
+#
+# from repository root. If dst-dir is not specified then configs are written
+# into the default directory (./configs).
 
 args="--dump_config"
+extra=
 
-
-for cmd in "${binaries[@]}"; do
- if [[ "$#" == "1" ]]; then
-    args="--dump_config --config_file=$1/acra-${cmd}.yaml"
- fi
- go run ./cmd/acra-${cmd}/*.go `echo ${args}`
+for cmd in $(ls ./cmd/ | grep ^acra-); do
+    if [[ $# == 1 ]]; then
+        extra="--config_file=$1/${cmd}.yaml"
+    fi
+    if [[ -f ./$cmd ]]; then
```

Comment:
can you add some comments for what it and where it used? because after several months someone may remove it as unnecessary piece of code)

Reply:
> can you add some comments for what it and where it used?

Sure, will do. You're right that a comment is more visible than `git blame`.

Reply:
Done. Now there is a comment explaining that this line exists to speed up test execution.